src/Header.js

```diff
@@ -61,6 +61,13 @@ class Header extends Component {
           </Link>
           <Subheader>Active Games</Subheader>
           <MenuItem primaryText="Not implemented yet" disabled={true} />
+          <Link style={{textDecoration: 'none'}} to="/dataViewer">
```

Comment:
Add a `<Subheader>` to separate this menu item from "Active Games", e.g. "Debug Tools".